src/main/webapp/components/workspace/workspace.js

```diff
@@ -279,6 +288,19 @@ export default () => {
                   onChange={filters => setFilters(filters)}
                 />
                 {Object.keys(status).length !== 0 ? <Divider /> : null}
+                {results.length > 0 && (
+                  <Pagination
+                    page={page}
+                    resultCount={
+                      Object.keys(status).length !== 0 &&
```

Comment:
These conditionals will change in the future depending on the resolution of https://github.com/connexta/revelio/issues/227